apps/web/src/services/project/components/ProjectMemberInviteModal.vue

```diff
@@ -189,41 +173,30 @@ const setExternalItemsInvalidTexts = async (userItem: InputItem): Promise<Extern
 };
 
 /* Api */
-const listRoles = async () => {
-    const { results } = await SpaceConnector.client.identity.role.list({
-        role_type: 'PROJECT',
-    });
-    state.roleItems = results.map((d) => ({
-        type: 'item',
-        label: d.name,
-        name: d.role_id,
-        pagePermissions: d.page_permissions,
-    }));
-};
 const addMember = async () => {
     try {
-        const params: any = {
-            role_id: selectedRoleItems.value[0].name,
+        const params: ProjectAddUsersRequestParams = {
+            project_id: props.projectId,
+            workspace_id: '', // TODO: workspace_id
```

Comment:
suggestion: I think it's not needed because workspace_id will be added by SpaceConnector level.